packages/content-management/content_insights/content_insights_public/src/components/views_stats/views_stats.tsx

```diff
@@ -86,7 +86,7 @@ const NoViewsTip = () => (
     content={
       <FormattedMessage
         id="contentManagement.contentEditor.viewsStats.noViewsTip"
-        defaultMessage="Views are counted everytime someone opens a dashboard (after version 8.16)"
```

Comment:
This change to a date might mislead on-prem users, no? If they don't upgrade to `8.16` in November but 6 months later?

Reply:
You're right... I've made a change, so we don't include a date